### DISCUSSION TITLE
This project is in stopped state and needs to be started first

### DISCUSSION THREAD

**Discussion by reubenwong97** [shuttle-hq/shuttle#1174] [Q&A]
Hi,

I have deployed a telegram bot to shuttle but I noticed that after a short while, I get the above message in the console. I could not find anything regarding auto-stopping of the service in the documentation. Does anyone know the cause, and a workaround for it?

Also, when I try to restart it following the prompt, I get:
```
❯ cargo shuttle project start
Error: 400 Bad Request
message: a project with the same name already exists
```

Thanks!

**Comment by oddgrd** [comment]
Hey @reubenwong97! You are likely running into this: https://docs.shuttle.rs/getting-started/idle-projects. We added a warning about this on project creation in the latest release. The reason for the 400 error is the project is idle, so to wake it you need to send a request to your app (which as stated in the docs above does not work for a telegram bot), or send a deployment command like `cargo shuttle deployment list`/`cargo shuttle status` etc.
